services/iam/app/controllers/devise/application_controller.rb

```diff
@@ -27,7 +27,10 @@ def user_resource
   end
 
   def tenant
-    Tenant.schema_name_from(account_id: sign_in_params[:account_id]) || Apartment::Tenant.current
+    return Tenant.schema_name_from(account_id: sign_in_params[:account_id]) if sign_in_params[:account_id].present?
+    return Tenant.find_by(alias: sign_in_params[:alias])&.schema_name if sign_in_params[:alias].present?
```

Comment:
The FE is only going to pass in the parameter `account_id`. It could have the value of either the `account_id` or the `alias` so both of these checks should use `sign_in_params[:account_id]`